lib/core/src/lib/form/components/widgets/core/form-field.model.ts

```diff
@@ -71,7 +75,7 @@ export class FormFieldModel extends FormWidgetModel {
     restLabelProperty: string;
     hasEmptyValue: boolean;
     className: string;
-    optionType: 'rest' | 'manual' | 'variable';
+    optionType: FieldOptionType;
```

Comment:
values should be either initialized here or in constructor or made optional (so something like `optionType?: FieldOptionType`)

Reply:
thx